src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1080,8 +1097,9 @@ internal void ReplaceSavedTokens(Token firstOldToken, Token lastOldToken, Token
             }
         }
 
-        private void ScanNewline(char c)
+        private void NormalizeCRLF(char c)
```

Comment:
What about `NormalizeEOL()`?

Reply:
Not specific enough. There are multiple possible end of lines. This method specifically normalizes CRLF line endings.